mirdata/datasets/freesound_one_shot_percussive_sounds.py

```diff
@@ -0,0 +1,298 @@
+        checksum="c67ce39d5aa6c6a7f88eedf7eb7d933e",
+    ),
+    "sound_info_analysis": download_utils.RemoteFileMetadata(
+        filename="sound_info_analysis.json",
+        url="https://zenodo.org/record/4687854/files/sound_info_analysis.json?download=1",
+        checksum="b51913a801bd59c2583d5f0e6f3c05b9",
+    ),
+    "metadata": download_utils.RemoteFileMetadata(
+        filename="licenses.txt",
+        url="https://zenodo.org/record/3665275/files/licenses.txt?download=1",
+        checksum="25f95a0e38d3ac4ae868f56c378fbccb",
+    ),
+    "readme": download_utils.RemoteFileMetadata(
+        filename="README.md",
+        url="https://zenodo.org/record/3665275/files/README.md?download=1",
+        checksum="afec91c033db607e2fc83c09940abd15",
+    ),
+}
+
+LICENSE_INFO = """
+The dataset is licensed under The Creative Commons Attribution Non Commercial Share Alike 4.0 International.
+Please check the specific license of each sound by running track.license
+"""
+
+
+class Track(core.Track):
+    """Freesound one-shot percussive sounds track class
+
+    Args:
+        track_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets/freesound_one_shot_percussive_sounds`
+
+    Attributes:
+        file_metadata_path (str): local path where the analysis file is stored and from where we get the file metadata
+        audio_path(str): local path where audio file is stored
+        track_id (str): track id
+        filename (str): filename of the track
+        username (str): username of the Freesound uploader of the track
+        license (str): link to license of the track file
+        tags (list): list of tags of the track
+        freesound_previews (dict): dict of Freesound previews of the track
+        freesound_analysis (str): dict of analysis parameters computed in Freesound using Essentia extractor
+        audiocommons_analysis (str): dict of analysis parameters computed using AudioCommons Extractor
+
+    Cached Properties:
+        file_metadata (dict): metadata parameters of the track file in form of Python dictionary
+
+    """
+
+    def __init__(
+        self,
+        track_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.file_metadata_path = self.get_path("analysis")
+        self.audio_path = self.get_path("audio")
+
+    @property
+    def tags(self):
+        return self._track_metadata.get("tags")
+
+    @property
+    def freesound_analysis(self):
+        return self._track_metadata.get("analysis")
+
+    @property
+    def audiocommons_analysis(self):
+        return self._track_metadata.get("ac_analysis")
+
+    @property
+    def freesound_previews(self):
```

Comment:
What is a preview, audio?

Reply:
maybe a url? If so, can we call it `freesound_preview_url`?

Reply:
yeah, it's a `dict` with the urls of the previews for the track. Updating it now!